### ISSUE TITLE
WARNING: One of the following dependencies is unresolved

### ISSUE THREAD

**Issue by cancan101** [uqfoundation/pathos#48]
After checking out master and then running `sudo python setup.py install`, I see:
```
***********************************************************
WARNING: One of the following dependencies is unresolved:
    pp >=1.6.4.4
    pyre ==0.8.2.0-pathos
    dill >=0.2.2
    pox >=0.2.1
    processing ==0.52-pathos
***********************************************************


Pathos relies on modified distributions of 'processing', 'pp', and 'pyre'.
Please download and install unresolved dependencies here:
  http://dev.danse.us/packages/
or from the "external" directory included in the pathos source distribution.
```

but as far as I can tell, everything is installed:
```
$ pip freeze | grep processing
processing==0.52-pathos
$ pip freeze | grep pp
pp==1.6.4.4
$ pip freeze | grep pox
pox==0.2.1
$ pip freeze | grep dill
dill==0.2.2
$ pip freeze | grep pyre
pyre==0.8.2.0-pathos
```

**Comment by mmckerns** [comment]
Yep, this is a known issue.  I'll see if I can fix it for the next release.

**Comment by mmckerns** [comment]
Should be fixed in: 5bf64cab32138641db6dcba6bd4601ea64df8954
